spec/unit/service_spec.rb

```diff
@@ -399,6 +399,15 @@ module Backend
           expect(Service.default_id(service.provider_key)).to eq '7002'
         end
 
+        it 'deletes all service errors' do
+          Service.save! id: 7002, provider_key: 'foo', default_service: true
+          ErrorStorage.store(service.id, ApplicationNotFound.new('foo'))
+
+          expect(ErrorStorage.count(service.id)).to be > 0
```

Comment:
With the pattern: `expect, do_something, expect`, I prefer to use `to_change`:
```ruby
expect { Service.delete_by_id(service.id) }
  .to_change(ErrorStorage.count(service.id))
  .from(2?)
  .to(0)
```

Reply:
I think you mean

```
expect { Service.delete_by_id(service.id) }.to  change { ErrorStorage.count(service.id) }.from(2?).to(0)
```